sympy/core/relational.py

```diff
@@ -186,6 +190,10 @@ def _eval_simplify(self, ratio, measure):
             if v is not None:
                 r = r.func._eval_relation(v, S.Zero)
 
+            numer, denom = fraction(simplify(dif))
```

Comment:
Is `simplify` needed here? It's already called on each side independently above. 

I think what you want here is `dif.as_numer_denom()`, which does some basic simplification (but nothing too expensive).